src/main/scala/com/yotpo/metorikku/metric/stepActions/Sql.scala

```diff
@@ -27,7 +27,7 @@ case class Sql(query: String, dataFrameName: String, showPreviewLines: Int,
   private def printStep(stepResult: DataFrame, stepName: String): Unit = {
     if (showPreviewLines > 0) {
       log.info(s"Previewing step: ${stepName}")
-      stepResult.printSchema()
+      log.debug("Schema:\n" + stepResult.schema.treeString)
```

Comment:
Why debug only?

Reply:
Because sometimes schemas are really large and they spam the log...

Reply:
Where are you previewing?
Locally? CI? production?
In any case it's good to put it in a logger, but I prefer info, since it's interesting in many times.
However, if you feel like it's not important enough I don't mind this will become debug, just make sure @doronporat is aligned, Thanks!